Create_Volume_From_Snapshot.md

```diff
@@ -0,0 +1,84 @@
+Create Volumes from Snapshots
+
+**Status**: Pending
+
+**Version**: Alpha
+
+**Author(s)**: Xiaoyan Li (@lixiaoy1)
+
+## Summary
+
+This proposal is drafted for creating a volume from snapshot supported backend storages. It defines the APIinterfaces to OpenSDS customers and meanwhile the interfaces for kinds of storage drivers.
+
+## Motivation
+
+Under some scenarios users may want to reuse the data on snapshots, with this function they can easily do it.
+
+### Goals
+
+* Design a unified API interface to create volumes from snapshots supported by backend storages.
+* Complete the common logic to create volumes from snapshots, and implementations in LVM, Ceph and Cinder.
+
+### Non-Goals
+
+* The implementations inside storage drivers except LVM, Ceph and Cinder.
+* It doesnot provide the function to create volumes from snapshots which isnot supported by backend storages, like to create volumes from snapshots in different backend storages.
```

Comment:
Missing space between "does" and "not".

Missing space between "is" and "not".

Suggest start a new sentence: For example,  creating volumes from snapshots in different backend storages is not supported.

Reply:
Done.